spec/storagefreak_spec.coffee

```diff
@@ -425,6 +425,42 @@ describe 'StorageFreak', ->
       @instance.search('as')
       expect(spy).to.not.be.called
 
+    it 'matches only infix data by default', (done)->
+      @instance._data = {
+        add: 'addas'
+        asd: 'asd'
+        as: 'as'
+      }
+
+      @instance.on 'dbsearch_results', (data, query, remote = false)->
```

Comment:
`query`, `remote` don't seem to be used in this anon function, you may remove them